src/models/Genre/GenreModel.ts

```diff
@@ -0,0 +1,40 @@
+import { Entry, EntrySkeletonType, FieldsType } from 'contentful';
+
+export type GenreContentfulEntry = Entry<EntrySkeletonType<FieldsType, string>, undefined, string>;
+
+export type Genre = {
+	id: string;
+	title: string | null;
+	movies: Array<GenreMovie>;
+};
+
+// Might want to add more fields here in the future
+export type GenreMovie = {
+	id: string;
+};
+
+export default class GenreModel {
```

Comment:
This model is not being used right now. Decided to keep it, but could just delete.
Since this endpoint is just reading from DB and not doing anything else, the model is not used.